src/test/java/org/kainos/ea/team2/integration/JobIntegrationTest.java

```diff
@@ -27,20 +28,33 @@ public class JobIntegrationTest {
             new ResourceConfigurationSourceProvider()
     );
 
+    private static final String VALID_USER_NAME = System.getenv("TEST_VALID_USERNAME");
+    private static final String VALID_USER_PASSWORD = System.getenv("TEST_VALID_PASSWORD");
+
+    private String getJWT() {
+        if(VALID_USER_NAME == null || VALID_USER_PASSWORD == null){
+            throw new IllegalArgumentException("Test credential environment variables not set!");
+        }
+        BasicCredentials credentials = new BasicCredentials(VALID_USER_NAME,VALID_USER_PASSWORD);
+        Response response = APP.client().target("http://localhost:8080/api/login").request().post(Entity.json(credentials));
+
+        return response.readEntity(String.class);
+    }
+
 
     /**
      * Verify that the getJobs method returns a list of jobs from the database.
      */
     @Test
     void getJobs_shouldReturnListOfJobs() {
+        String jwt = getJWT();
 
         // list of employees, add each employee returned from the db
         List<Job> response = APP.client().target("http://localhost:8080/api/job-roles")
-                .request().get(List.class);
+                .request().header("Authorization", "Bearer " + jwt).get(List.class);
```

Comment:
can you add integration tests for each endpoint without this header and verify unauthorized returned?